src/Components/Notes/NoteCard.jsx

```diff
@@ -213,16 +250,27 @@ const ShowMore = ({onClick, expandText}) => {
 }
 
 
-const CardActions = ({
+const CardFooter = ({
+  id,
+  issueNumber,
+  username,
   onClickCamera,
   onClickShare,
   numberOfComments,
   selectCard,
   embeddedCameras,
-  selected}) => {
+  selected,
+  deleteNote,
+  isComment,
+  synchedNote}) => {
   const [shareIssue, setShareIssue] = useState(false)
+  const repository = useStore((state) => state.repository)
+  const toggleSynchNotes = useStore((state) => state.toggleSynchNotes)
```

Comment:
The logic around synchronization of notes should be refactored to take into account that there is no "delay" anymore.

Reply:
I was working in the library today, and the connection was rather slow, it felt much smoother with synching.